src/components/switch/switch.scss

```diff
@@ -2,8 +2,14 @@
 
 @import '@limetech/mdc-switch/mdc-switch';
 
+:host {
+    display: inline-flex;
+    align-items: center;
+}
+
 .mdc-switch {
     @include mdc-switch-toggled-on-color(primary);
+    line-height: normal;
```

Comment:
Was this intended to go on the label? I've only checked in Chrome on macOS, but switching this on and off makes no difference, at least not where I'm checking. I tried adding `line-height:normal` to the label though, and that changed the alignment a pixel or two, the result of which I thought looked better.

Reply:
awesome catch! yes, it was for the label 😅